### PR TITLE
Add Practical Go lessons

### LINKED ISSUE #780: [Add] <practical-go-lessons site in GO subcategory to learn GO from scratch>
### Site URL

https://www.practical-go-lessons.com/

### Description of the above provided site

Practical GO lessons is a site to learn GOlang for beginners through docs and videos.

### Category

Languages

### Sub Category

GO

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Ayaan49**
## Fixes Issue
 -->  Closes #780 

## Changes proposed
 --> Added link to Go subcategory and Languages category.

### CODE REVIEW

File: database/languages/golang.json [hunk 1]
```diff
@@ -10,7 +10,14 @@
     "name": "Hitesh Choudhary",
     "description": "This playlist contains all the concepts of Golang along with two mini projects",
     "url": "https://youtube.com/playlist?list=PLRAV69dS1uWQGDQoBYMZWKjzuhCaOnBpa",
-    "category": "youtube",
+    "category": "Languages",
+    "subcategory": "golang"
+  },
+  {
+    "name": "Practical Go Lessons",
+    "description": "Practical GO lessons is a site to learn GOlang for beginners through docs and videos.",
```
Comment (CBID2) [inline database/languages/golang.json:18]:
```suggestion
    "description": "This site provides beginners-level GOlang lessons through docs and videos.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just need to fix this proofreading era.

**Comment by Ayaan49** [comment]
> Just need to fix this proofreading era.

Done

**Review by ujjawaltyagii — APPROVED** [review]
Lgtm 👍

File: database/languages/golang.json [hunk 2]
```diff
@@ -10,7 +10,14 @@
     "name": "Hitesh Choudhary",
     "description": "This playlist contains all the concepts of Golang along with two mini projects",
     "url": "https://youtube.com/playlist?list=PLRAV69dS1uWQGDQoBYMZWKjzuhCaOnBpa",
-    "category": "youtube",
+    "category": "Languages",
```
Comment (rupali-codes) [inline database/languages/golang.json:13]:
```suggestion
    "category": "languages",
```

File: database/languages/golang.json [hunk 3]
```diff
@@ -10,7 +10,14 @@
...
+    "subcategory": "golang"
+  },
+  {
+    "name": "Practical Go Lessons",
+    "description": "This site provides beginners-level GOlang lessons through docs and videos.",
+    "url": "https://www.practical-go-lessons.com/",
+    "category": "Languages",
```
Comment (rupali-codes) [inline database/languages/golang.json:20]:
```suggestion
    "category": "languages",
```

**Review by CBID2 — APPROVED** [review]
Great job @Ayaan49! 😄 LGTM! 😄

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**